### PR TITLE
Setting an owners file.

### PR DESCRIPTION
**Pull request by hh**
```release-note
Added initial OWNERS file to assist in automation
```

### CODE REVIEW

**Comment by devaii** [comment]
Awesome. This will be nice to have working.

Add [release notes](https://github.com/kubernetes/community/blob/master/contributors/guide/release-notes.md), I suppose.

/lgtm

**Comment by hh** [comment]
```release-note
Added initial OWNERS file to assist in automation
```

**Comment by hh** [comment]
@zachmandeville can you add a final review then /approve ?

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    labels:
+      - lang/python
+  "*\\.js$":
+    approvers:
+      - zachmandeville
+    reviewers:
+      - hh
+      - devaii
+    labels:
+      - lang/javascript
```
[the remark this answers is not in the document]
Comment (hh) [reply OWNERS]: Spaces are hard to edit (especially via the web).
I pulled this down and was able to make the change locally.

**Comment by hh** [comment]
@devaii can you /approve?

**Comment by hh** [comment]
I'm unsure what we needed to do to get @cncf-ci to merge this.
It may be that it wanted the master/OWNERS file to reflect who could approve.

**Pull request merged**